<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QPushButton" name="btnClearContents">
          <property name="text">
           <string>清空表格内容</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="btnIniData">
          <property name="text">
           <string>初始化表格数据</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnAppendRow">
          <property name="text">
           <string>添加行</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="btnAutoHeight">
          <property name="text">
           <string>自动调节行高</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="btnSetHeader">
          <property name="text">
           <string>设置表头</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btnDelCurRow">
          <property name="text">
           <string>删除当前行</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinRowCount"/>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btnInsertRow">
          <property name="text">
           <string>插入行</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnSetRows">
          <property name="text">
           <string>设置行数</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="chkBoxRowColor">
          <property name="text">
           <string>间隔行底色</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="chkBoxHeaderH">
          <property name="text">
           <string>显示行表头</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btnAutoWidth">
          <property name="text">
           <string>自动调节列宽</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="chkBoxEditable">
          <property name="text">
           <string>表格可编辑</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="chkBoxHeaderV">
          <property name="text">
           <string>显示列表头</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btnReadToText">
          <property name="text">
           <string>读取表格内容到文本</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="radioSelectItem_2">
          <property name="text">
           <string>行选择</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QRadioButton" name="radioSelectItem">
          <property name="text">
           <string>单元格选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableInfo">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
        <property name="rowCount">
         <number>5</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>列1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>列2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>列3</string>
         </property>
        </column>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string>0行，0列</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
       <widget class="QPlainTextEdit" name="textEdit"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
